api/src/pcapi/connectors/serialization/boost_serializers.py

```diff
@@ -49,46 +49,39 @@ def _convert_to_utc_datetime(datetime_with_tz_offset: datetime.datetime) -> date
     return datetime_with_tz_offset.astimezone(tz=datetime.timezone.utc)
 
 
-class ShowTime4(BaseModel):
-    """We transcribe their API schema and keep their name convention"""
-
-    id: int
-    numberSeatsRemaining: int
-
-
-class ShowTimeCollection(Collection):
-    data: list[ShowTime4]
-
-
 class ShowtimePricing(BaseModel):
     id: int
     pricingCode: str
     amountTaxesIncluded: Decimal
     title: str
 
 
-class ShowTime(BaseModel):
+class ShowTime4(BaseModel):
+    """We transcribe their API schema and keep their name convention"""
+
     id: int
     numberSeatsRemaining: int
     showDate: datetime.datetime
     showEndDate: datetime.datetime
-    soldOut: bool
-    authorizedAccess: bool
     film: Film2
     format: dict
     version: dict
     screen: dict
-    showtimePricing: list[ShowtimePricing]
+    showtimePricing: list[ShowtimePricing] | None
```

Comment:
```suggestion
    showtimePricing: list[ShowtimePricing] = pydantic.v1.Field(default_factory=list)
```

Comme ça on a pas besoin de ce trimballer du typage `something | None` un peu partout ni de ce [bloc conditionnel](https://github.com/pass-culture/pass-culture-main/pull/8432/files#diff-c94afb440ebe5709324ea6881d33f1b5e3cf8e8137dae7eccdd58004e0366467R26)

(Dans l’esprit de cette [MR](https://github.com/pass-culture/pass-culture-main/pull/8324))